Formula/chapel.rb

```diff
@@ -60,9 +69,193 @@ def install
 
   test do
     ENV["CHPL_HOME"] = libexec
+    ENV["CHPL_INCLUDE_PATH"] = HOMEBREW_PREFIX/"include"
+    ENV["CHPL_LIB_PATH"] = HOMEBREW_PREFIX/"lib"
     cd libexec do
-      system "util/test/checkChplInstall"
+      with_env(CHPL_LLVM: "system") do
+        system "util/test/checkChplInstall"
+      end
+      with_env(CHPL_LLVM: "none") do
+        system "util/test/checkChplInstall"
+      end
     end
     system bin/"chpl", "--print-passes", "--print-commands", libexec/"examples/hello.chpl"
   end
 end
+
+__END__
```

Comment:
Yes, they are in the work-in-progress PR https://github.com/chapel-lang/chapel/pull/18880 which I am expecting to complete soon along with some other changes that will improve portability

Reply:
@SMillerDev: For a bit more context on the patches: We were caught by surprise by the CI checks in some cases with this PR because we don't have an M1 Mac in-house to do testing on (though we're working on addressing that) and didn't have problems with the M1 CI checks in our recent 1.25.0 release.  So until we opened up the PR and started seeing the failures due to the stricter M1 checks, we were unaware of the issues.  I think one of the others may have been due to the change in this formula to start using homebrew GMP rather than our bundled version, previously requested by @carlocab, which revealed some portability challenges due to different paths on the non-vanilla Mac builds.  As Michael says, we're working on incorporating changes into our `main` branch to avoid the need for these in future releases, but are hoping to proceed with this PR in the meantime to catch homebrew up with our other 1.25.1 releases.

Reply:
That's okay, just wanted to ensure this wasn't meant to hang around in Homebrew forever.